docs/concepts/trading-on-vega/positions-margin.md

```diff
@@ -3,47 +3,42 @@ sidebar_position: 2
 title: Positions and margin
 hide_title: false
 ---
-Trading margined derivatives (such as futures) allows you to create leveraged positions, meaning you only need some of the notional value of a trade to place orders on a market on Vega. 
+Trading margined derivatives (such as futures) allows you to create leveraged positions, meaning you only need some of the notional value of an instrument to place orders on a market on Vega. 
 
-Leverage is, in effect, the inverse of the initial margin that the protocol takes from your collateral when you place a trade. 
```

Comment:
I don't mind if you reject this sentence specifically, but I did want to include at least a nod to the question of "how does leverage work on Vega?" so I'd be very keen for an alternative :)

Reply:
It's just that it's inaccurate, and I though "Leverage, meanwhile, describes how many times larger is the notional value of that position compared to the margin you have dedicated to it." below does the trick.

Reply:
Great, that works, thank you!